headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -75,6 +79,23 @@ struct move {
 };
 ASSERT_SIZE(struct move, 8);
 
+// Used to hold data during damage calculation
```

Comment:
Is this the thing that's initialized by `ResetDamageCalcScratchSpace`? If so we should probably update the relevant functions in a follow-up commit (we don't have to do it now though).

Reply:
No, this is a smaller struct kept in the stack. Looks like the one initialized by that function is a different struct (0x54 bytes long) that is stored in the dungeon struct (offset 0x184).

Reply:
Okay, disregard then.